src/routing/pages/probe/Explore.svelte

```diff
@@ -21,6 +22,7 @@
 
 <Probe let:data let:probeType>
   <div in:fade class="graphic-body__content">
+    <ProbeTitle view={$store.route.view} probeName={$store.probe.name} />
```

Comment:
Do we think there will be cases where `view` is equal to something other than `$store.route.view`? If not, would it make sense to consider not taking this as a prop at all, but instead accessing `$store.route.view` from the `ProbeTitle` component?